src/pykiso/lib/auxiliaries/udsaux/uds_auxiliary.py

```diff
@@ -114,9 +114,11 @@ def send_uds_raw(
             False
         """
         try:
-            log.internal_info(
-                f"UDS request to send '{['0x{:02X}'.format(i) for i in msg_to_send]}'"
-            )
+            if log.isEnabledFor(logging.getLogger().level):
```

Comment:
IMO this should be tackled in a separate task to have a proper filtering of our internal log levels as for any other default log level

Reply:
How does the user turn off/on the internal logs ?

I would have thought that he can control it via the uds_aux level. Or via decorator ?

Reply:
Actually internal logs can only be off if the user choose a level higher than INFO for log level of the test which is why I added a decorator `disable_logging` to be able to disable logging for a specific test

Reply:
The disable_logging is totally fine, it's more regarding the workarounds you had to do to avoid the internal log level messages to be evaluated
This should then be fixed in logging_initializer, where the internal log levels should always be lower than the non-internal ones so that the logging module can filter them out by itself